packages/kit/src/exports/vite/index.js

```diff
@@ -348,6 +355,7 @@ async function kit({ svelte_config }) {
 					__SVELTEKIT_EMBEDDED__: s(kit.embedded),
 					__SVELTEKIT_EXPERIMENTAL__REMOTE_FUNCTIONS__: s(kit.experimental.remoteFunctions),
 					__SVELTEKIT_CLIENT_ROUTING__: kit.router.resolution === 'client' ? 'true' : 'false',
+					__SVELTEKIT_SERVER_TRACING_ENABLED__: kit.experimental.tracing.server ? 'true' : 'false',
```

Comment:
i know we use both versions of this for some inexplicable reason but NOT FOR THE SAME DANG VARIABLE

```suggestion
					__SVELTEKIT_SERVER_TRACING_ENABLED__: s(kit.experimental.tracing.server),
```